browser-test/clear_data.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+import os
+import sys
+import json
+
+project_path = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
+browser_test_path = os.path.join(project_path, 'browser-test')
+test_data_file = open(os.path.join(browser_test_path, 'testData.json'))
+test_data = json.loads(test_data_file.read())
+test_data_file.close()
+
+OPENWISP_RADIUS_PATH = os.getenv('OPENWISP_RADIUS_PATH', '')
+
+if OPENWISP_RADIUS_PATH == '':
+    print('OPENWISP_RADIUS_PATH is needed for initializing data.')
```

Comment:
If I do not set `$OPENWISP_RADIUS_PATH`, I get:

```
TimeoutError: Waiting element to become stale
Wait timed out after 5084ms
```

(I lowered the timeout to 5 secs as mentioned in my prev comment)

I was expecting to see this error message, but that's not happening, so this part of the code is not working as expected.

Please ensure the error message is shown if the variable is not set.